wa/commands/show.py

```diff
@@ -98,3 +105,9 @@ def get_rst_from_target(target):
     text += get_params_rst(target.assistant_params)
     text += '*Note:* For available runtime parameters please see the documentation'
     return text + '\n'
+
+def get_rst_for_global_config():
+    text = underline('Global Configuration')
```

Comment:
I think it might be useful to add a short paragraph here explaining what this. I.e. That this is controls the behaviour of WA/run as a whole and that it should go into a config file (either in `$WA_USER_DIRECTORY/config.yaml` or one specified with `-c`), or into  `config`/`global` section of the agenda.